packages/react-components/src/components/ImageGallery.tsx

```diff
@@ -104,17 +103,16 @@ export interface ImageGalleryStrings {
  */
 export const ImageGallery = (props: ImageGalleryProps): JSX.Element => {
   const { isOpen, images, onImageDownloadButtonClicked, onDismiss, onError, startIndex = 0 } = props;
-  const theme = useTheme();
-  const isDarkTheme = isDarkThemed(theme);
+  const theme = useTheme() as unknown as ChatTheme;
```

Comment:
This cast should not be needed; does the useTheme() hook type need updating?

Reply:
Chatted offline. We'll keep the current way of casting until Fluent 9 migration